client/packages/invoices/src/Returns/InboundDetailView/DetailView.tsx

```diff
@@ -74,6 +88,7 @@ export const InboundReturnDetailView: FC = () => {
               returnId={data.id}
               initialItemId={itemId}
               modalMode={mode}
+              loadNextItem={nextItemId ? () => onOpen(nextItemId) : undefined}
```

Comment:
Made this function's existence conditional, as otherwise we'd need another prop just to tell the modal whether or not there even *is* a next item.